scripts/agent.yml

```diff
@@ -211,3 +219,20 @@ buildvariants:
       - localhost
     tasks:
       - name: my_group
+  - name: container_bv
+    display_name: container_bv
+    run_on:
+      - container
+    tasks:
+      - name: container_task
+
+containers:
+  - name: container
+    working_dir: /
+    image: "hadjri/evg-e2e-test-ubuntu"
```

Comment:
Can we make a TODO to eventually use a more official image repo, like one in our AWS account's ECR repository? I'm not sure if this configuration value even matters since the pod and pod document are manually populated in the smoke test data anyways.